docs/reference/ml/df-analytics/apis/get-dfanalytics-stats.asciidoc

```diff
@@ -477,7 +477,17 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=node-transport-address]
 =====
 `phase`::::
 (string) Defines the phase of the {dfanalytics-job}. Possible phases: 
-`reindexing`, `loading_data`, `analyzing`, and `writing_results`.
+* `reindexing`, 
+* `loading_data`, 
+* `computing_outliers` (for {oldetection} only),
+* `feature_selection` (for {regression} and {classification} only),
+* `coarse_parameter_search` (for {regression} and {classification} only),
+* `fine_tuning_parameters` (for {regression} and {classification} only),
+* `final_training` (for {regression} and {classification} only),
+* `writing_results`.
+
```

Comment:
I think you need a continuation character here to get the following paragraph nested properly:
```suggestion
+
```